pixelelated: replace the default splash and its display name (#337).

Keep the distribution:rocknix setting and custom splash path compatible.

--- a/theme.xml
+++ b/theme.xml
@@ -23,7 +23,7 @@
    -->
    <subset name="distribution" displayName="Distribution">
       <include name="batocera" displayName="Batocera/Knulli" />
-      <include name="rocknix" displayName="RockNIX" />
+      <include name="rocknix" displayName="pixelelated" />
       <include name="retrobat" displayName="Retrobat" />
    </subset>
 
--- a/splash.xml
+++ b/splash.xml
@@ -27,8 +27,11 @@
          <path>${themeCustomizationsPath}splash.jpg</path>
       </image>
       <image ifSubset="splash-screen:default" name="background">
-         <origin>0 0</origin>
-         <pos>1 1</pos>
+         <origin>0.5 0.5</origin>
+         <pos>0.5 0.36</pos>
+         <maxSize>0.8 0.6</maxSize>
+         <path>./pixelelated-wordmark.svg</path>
+         <linearSmooth>false</linearSmooth>
       </image>
       <image name="progressbar">
          <origin>0.5 0.5</origin>
